packages/compass-e2e-tests/helpers/compass.ts

```diff
@@ -949,18 +949,20 @@ async function getCompassBuildMetadata(): Promise<BinPathOptions> {
 }
 
 export async function buildCompass(
-  force = false,
   compassPath = COMPASS_DESKTOP_PATH
```

Comment:
yes the second parameter became the first, but we never pass either of these parameters, so that didn't require other changes.